browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -114,5 +128,7 @@ void BraveAppearanceHandler::OnPreferenceChanged(const std::string& pref_name) {
   if (IsJavascriptAllowed()) {
     FireWebUIListener("super-referral-active-state-changed",
                       base::Value(IsSuperReferralActive(profile_)));
+    FireWebUIListener("binance-supported-state-changed",
```

Comment:
Firing `binance-supported-state-changed` message here doesn't make sense because `OnPreferenceChanged` is called only when `kNewTabPageSuperReferralThemesOption` pref is changed.
If binance supported state is changed during the runtime, `BraveAppearanceHandler` should monitor its state and it should only fire this message when that state is changed.

If binance supported state is not changed in runtime now, I think we don't need this message.

Reply:
good call, supported state doesn't change during runtime. removed this and the client side listener